.github/ISSUE_TEMPLATE/contribution_bug.yaml

```diff
@@ -0,0 +1,54 @@
+name: Report a contribution bug
```

Comment:
That name is too vague. What's a contribution bug? Maybe "Report a process bug" is better? Let's discuss.

Reply:
What's a process bug? 😄 @brentholtorf could you help us come up with a good name? 

Background: 
We want a new issue template that allows for reporting issues discovered in the development/contribution process to BCApps. For example, it someone can't set up an dev environment, our documentation isn't accurate etc. Things that block/make it harder for developers to contribute to BCApps